### ISSUE TITLE
Obstacle loss function

### ISSUE THREAD

**Issue by msinto93** [talsperre/INFER#2]
In the paper you say that you add a safety loss term, that penalizes all predicted
states of vehicles that lie in an obstacle cell. In the args.txt file supplied with the pretrained KITTI model, `--lossOT = True`, suggesting you have used this loss function. However, in the training code you define the `obstacleLossFun` then never use it.

This is just one example of a load of seemingly unused args, making it very difficult to recreate your results given that it is not clear exactly which args you used in your training.

**Comment by abhikandoi2000** [comment]
It seems the function is only defined and never used. How did you guys use this loss function?
We are trying to reproduce your results @talsperre

**Comment by fgqile** [comment]
> It seems the function is only defined and never used. How did you guys use this loss function? We are trying to reproduce your results @talsperre

Hi, have you reproduced this code?  I am trying to reproduce the results too. but I don't have the same results.

**Comment by fgqile** [comment]
> It seems the function is only defined and never used. How did you guys use this loss function? We are trying to reproduce your results @talsperre

Hi, have you reproduced this code?  I am trying to reproduce the results too. but I don't have the same results.
